pact/coin-contract/v2/coin.pact

```diff
@@ -48,6 +48,10 @@
     "Magic capability constraining genesis transactions"
     true)
 
+  (defcap SLASH ()
```

Comment:
We might as well bring in the new Pact and make this a auto-managed cap with the amount